admin/module/readLinkMaintenance.py

```diff
@@ -0,0 +1,131 @@
+
+'''
+@author      Stephen White  <stephenawhite57@gmail.com>
+@copyright   Stephen White
+@link        https://github.com/readsoftware
+@version     1.0
+@license     <http://www.gnu.org/licenses/>
+@package     READ.Admin.Module
+READ link Maintenance Library
+'''
+import readQueryCursor as rqc
+
+linkColTableLookupByPrefix = {
+    'seq':['sequence','seq_entity_ids'],
+    'cmp':['compound','cmp_component_ids'],
+    'inf':['inflection','inf_component_ids'],
+    'lem':['lemma','lem_component_ids'],
+    'edn':['edition','edn_sequence_ids'],
+    'tok':['token','tok_grapheme_ids'],
+    'scl':['syllablecluster','scl_grapheme_ids'],
+    'ano':['annotation','ano_linkto_ids'],
+}
+
+# TODO: create api for connect to db
+
+def getChildLinksForContainer(containerGID):
+    if containerGID == None:
+        print(f'call to getChildLinkdsForContainer with invalid param {containerGID}')
+        return []
+    if containerGID.find(':') != 3:
+        print(f'call to getChildLinkdsForContainer expected GID to have : {containerGID}')
+        return []
+    prefix,entID = containerGID.split(":")
+    useGIDForm = False
+    if not prefix in linkColTableLookupByPrefix.keys():
+        print(f'call to getChildLinkdsForContainer unexpected prefix "{prefix}"')
+        return []
+    else:
+        tblName, colName = linkColTableLookupByPrefix.get(prefix)
+        myRQC.query(f"select {colName} from {tblName} where {prefix}_id = {entID};")
+        return myRQC.getRow(0)[0]    
```

Comment:
Maybe I am missing, but I don't understand how you can query like this. Shouldn't you at least pass a ReadQueryCursor parameter in the function?